src/main/java/webApp/QuestionMessage.java

```diff
@@ -0,0 +1,20 @@
+package webApp;
+
+public class QuestionMessage {
+
+    private final String type;
+    private final String question;
```

Comment:
Wondering if it would make any sense to have both these constants as "static"? Can someone key in an see if makes any sense

Reply:
Could almost be an enum too?